src/utils/tree.ts

```diff
@@ -31,11 +31,22 @@ export type TreeOptions = {
    * @default "  "
    */
   prefix?: string;
+  /**
+   * Limit the depth of tree
+   */
+  depth?: number;
```

Comment:
Maybe call it maxDepth?

Reply:
maxDepth sounds sensible to me.  It would be fantastic if consola added similar functionality on the actual output of a consola.log [being too long](https://github.com/unjs/consola/issues/246). 😄